website/src/pages/index.tsx

```diff
@@ -19,16 +20,16 @@ function HomepageHeader() {
                 <Heading as="h1" className="hero__title">
                     {siteConfig.title}
                 </Heading>
-                <p className="hero__subtitle">{siteConfig.tagline}</p>
+                <p className="hero__subtitle"><Translate>Execute and interact with your Markdown Python Codeblocks in Docusaurus 3</Translate></p>
                 <div className={styles.buttons}>
                     <Link className="button button--lg button--success" to={withBaseUrl('docs')}>
-                        Get Started
+                        <Translate>Get Started</Translate>
                     </Link>
                     <Link className="button button--secondary button--lg" to={withBaseUrl('/demo')}>
-                        Live Demo
+                        <Translate>Live Demo</Translate>
                     </Link>
                     <Link className="button button--secondary button--lg" to={withBaseUrl('/snippets')}>
-                        Share Code
+                        <Translate>Share Code</Translate>
```

Comment:
i prefer explicit id attributes here - otherwise the translation files need to be changed when e.g. a typo needs to be fixed.